graphql/tests/projectSettings/results.json

```diff
@@ -119,8 +119,9 @@
         "data": {
           "projectSettings": {
             "vars": {
-              "vars": {"hello": "", "foo":  "bar"},
-              "privateVars": ["hello"]
+              "vars": {"hello": "", "foo":  "bar", "admintest": "woo"},
+              "privateVars": ["hello"],
+              "adminOnlyVars": ["admintest"]
```

Comment:
@ablack12 Do you know if we already have a UI ticket to add this to the variables page?

Reply:
oh never mind, that work is in the linked PR :)